nautobot_ssot_infoblox/diffsync/models/nautobot.py

```diff
@@ -122,8 +262,20 @@ def create(cls, diffsync, ids, attrs):
             group=OrmVlanGroup.objects.get(name=ids["vlangroup"]) if ids["vlangroup"] else None,
             description=attrs["description"],
         )
-        _vlan.tags.add("SSoT Synced to Infoblox")
-        _vlan.validated_save()
+        if "ext_attrs" in attrs:
+            cls.process_ext_attrs(diffsync=diffsync, vlan=_vlan, extattrs=attrs["ext_attrs"])
+        _vlan.tags.add(create_tag_sync_from_infoblox())
+        # ensure that the VLAN Group and VLAN have the same Site
+        if not _vlan.site and _vlan.group.site:
+            _vlan.site = _vlan.group.site
+        if not _vlan.group.site and _vlan.site:
+            _vlan.group.site = _vlan.site
+            _vlan.group.validated_save()
```

Comment:
Same as below. It can be called below with the error handling.

Reply:
See comment about the other portion that's like this.